pootle/tools/createpootlepot

```diff
@@ -17,7 +17,7 @@ POOTLE_JS_POT="$POT_DIR/pootle_js.pot"
 
 # Header settings
 PACKAGE="Pootle"
-VERSION=`egrep "^ver" $SRC_DIR/__version__.py | sed "s,.*(\(.*\)),\1,g" | sed "s,[ ],,g" | sed "s/,/./g"`
+VERSION=$(python -c "from pootle.core.utils.version import *; print(get_main_version())")
```

Comment:
Had to make it slightly convoluted as we're just using the main version to avoid needless header noise.  But managed to craft a __main__ that allows you to ask for alternative forms as needed.